src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritServer.java

```diff
@@ -24,26 +24,11 @@
  */
 package com.sonyericsson.hudson.plugins.gerrit.trigger;
 
+import static com.sonyericsson.hudson.plugins.gerrit.trigger.utils.StringUtil.PLUGIN_IMAGES_URL;
```

Comment:
even if your IDE suggests it, please do not auto-format the files or change the order of imports. It makes your changes less visible and decreases your changes to merge them (believe me, I tried :) )

Reply:
(outdated)